src/kirby/components/segmented-control/segmented-control.component.tns.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, EventEmitter, Output, Input } from '@angular/core';
+
+import { SegmentItem } from './segment-item';
+
+@Component({
+  selector: 'kirby-segmented-control',
+  templateUrl: './segmented-control.component.html',
+  styleUrls: ['./segmented-control.component.scss'],
+})
+export class SegmentedControlComponent {
+  @Output() segmentClicked: EventEmitter<any> = new EventEmitter();
```

Comment:
Consider narrowing the emitted type to `string`, and add a type-doc comment mentioning that the id of the clicked segment is emitted.